nginx/assets-server-section.conf

```
@@ -0,0 +1,7 @@
+location ~ \.(css|js|png|svg) {
```

Would add a `$` at the end of the regex:

```suggestion
location ~ \.(css|js|png|svg)$ {
```

Might be nice to add json too (for the manifest).